package.json

```diff
@@ -18,61 +18,61 @@
     },
     "private": false,
     "dependencies": {
-        "@angular/platform-server": "^6.1.9",
+        "@angular/cdk": "^7.2.0",
+        "@angular/platform-server": "^7.1.4",
         "@ctrl/tinycolor": "^2.0.0",
         "angular2-hotkeys": "^2.1.2",
         "angular2-tinymce": "^2.1.2",
-        "core-js": "^2.5.3",
+        "core-js": "^2.5.4",
         "font-awesome": "^4.7.0",
         "lodash": "^4.17.11",
         "md5": "^2.2.1",
         "moment": "^2.19.4",
         "ng2-ace-editor": "^0.3.8",
         "ng2-dragula": "^2.1.0",
         "primeicons": "^1.0.0-beta.10",
```

Comment:
There is a final 1.0.0 version of primeicons.

Reply:
thanks to the `^` even when declared as `1.0.0-beta` it'll get the most recent major version 
https://stackoverflow.com/a/22345808

*Anyway I updated it